Sources/StreamChat/ChatClient.swift

```diff
@@ -189,9 +189,20 @@ public class ChatClient {
     
     /// Stream-specific request headers.
     private let sessionHeaders: [String: String] = [
-        "X-Stream-Client": "stream-chat-swift-client-v\(SystemEnvironment.version)"
+        "X-Stream-Client": "stream-chat-\(sdkIdentifier)-client-v\(SystemEnvironment.version)"
     ]
     
+    /// Identifies which SDK is being used.
+    private static var sdkIdentifier: String {
+        #if canImport(StreamChatSwiftUI)
+        return "swiftui"
+        #elseif canImport(StreamChatUI)
+        return "uikit"
+        #else
+        return "swift"
+        #endif
+    }
```

Comment:
Tricky question: What if a customer uses both `StreamChatSwiftUI` and `StreamChatUI` ? 🤔 Is that even possible?  Is this something we want to support? Or SwiftUI SDK will have totally feature parity with the UIKit SDK?

Reply:
The plan is to have feature parity with the UIKit SDK. In general, it will not make sense for both of them to be used in the same app, unless some gradual migration is done. But in those cases, the "swiftui" will be returned, which is somehow correct.

Reply:
if you use both than we should still track swiftui